jms/1.7/modules/ROOT/pages/jms-performance.adoc

```diff
@@ -1,47 +1,71 @@
-= JMS Tuning For Performance
+= Tune JMS Connector for Performance
 :keywords: jms, connector, consume, message, source, listener
 :page-aliases: connectors::jms/jms-performance.adoc
 
+Performance can be a factor when processing the loading of messages. The following recommendations can help you get the most of Anypoint Connector for JMS (JMS Connector) for your Mule applications.
 
+== Do not Disable Connections Caching
 
-Performance can be a big factor when processing loads of messages, which means that you probably want to check the following configurations to make sure you get the most out of the JMS Connector.
+Connections are expensive to create, therefore reuse them as much as you can. By default, JMS Connector uses a caching strategy configuration that reuses as many consumers and producers as possible. Do not disable connection caching because modifying this configuration can reduce the performance of the application.
```

Comment:
```suggestion
Because connections are expensive to create, reuse them as much as you can. By default, JMS Connector uses a caching strategy that reuses as many consumers and producers as possible. To avoid performance degradation, do not disable connection caching.
```